changelogs/unreleased/gh-7894-export-iproto-constants-and-features.md

```diff
@@ -0,0 +1,5 @@
+## feature/box
+
+* Exported IPROTO constants and features to Lua, and added a new
+  `IPROTO_UNKONWN` command code which will be used for overriding the unknown
+  request handler (gh-7894).
```

Comment:
Documenting, mentioning in changelog, or even adding `IPROTO_UNKNOWN` now doesn't make sense, because there's no such thing as `box.iproto.override` yet. Please drop it from this patch and instead add it in the patch that implements `box.iproto.override`.

Reply:
Fixed: dropped this commit.

Reply:
Sorry, overlooked it. Changed the doc bot request to document references.